src/promptflow/promptflow/integrations/langchain.py

```diff
@@ -11,12 +11,18 @@
 
 
 class PromptFlowCallbackHandler(BaseCallbackHandler):
+    """`PromptFlowCallbackHandler` implements the `BaseCallbackHandler` interface,
+    which has a method for each event that can be subscribed to.
```

Comment:
`~promptflow.xxx.BaseCallbackHandler`, and for other places

Reply:
`BaseCallbackHandler` is defined in `langchain` package, so I used the whole path `langchain.callbacks.base.BaseCallbackHandler` here.